scripts/run-artifactory.sh

```diff
@@ -3,7 +3,7 @@
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" > /dev/null && pwd )"
 source "${SCRIPT_DIR}/get-access-key.sh"
 source "${SCRIPT_DIR}/wait-for-rt.sh"
-export ARTIFACTORY_VERSION=${ARTIFACTORY_VERSION:-7.46.3}
+export ARTIFACTORY_VERSION=${ARTIFACTORY_VERSION:-7.49.8}
```

Comment:
When and why do we bump Artifactory version in these scripts?

Reply:
When we remember to :) and because we prefer to test against the latest released version so in case we missed any changes.

Reply:
so we update when we can, but we don't set the tag to latest in case there are breaking changes we don't know about, or so that we don't get caught off guard?

Reply:
This is the default, fallback version. You can export whatever version you wish to test against in your shell.

Reply:
This script is used only for manual testing. CI is always using the latest.

Reply:
Ideally this should be tracking the latest self-hosted version automatically 😄

Reply:
We can do it, but it will require Zscaler to get the latest versions from the versions endpoint in JSON, or Helm to get the latest version from the Helm chart repo.